pylint/checkers/variables.py

```diff
@@ -2024,16 +2024,17 @@ def _store_type_annotation_names(self, node):
 
     def _check_self_cls_assign(self, node):
```

Comment:
```suggestion
    def _check_self_cls_assign(self, node: nodes.Assign) -> None:
```
While we are at it, it makes sense to also add type annotations.